v2/internal/controllers/crd_aks_managedcluster_20230102preview_test.go

```diff
@@ -162,3 +169,53 @@ func AKS_ManagedCluster_Kubeconfig_20230102Preview_Secrets(tc *testcommon.KubePe
 	tc.PatchResourceAndWait(old, cluster)
 	tc.ExpectSecretHasKeys(secret, "admin", "user")
 }
+
+func AKS_ManagedCluster_TrustedAccessRoleBinding_20230102Preview_CRUD(
```

Comment:
Consider having this as a Subtest within `Test_AKS_ManagedCluster_20230202Preview_CRUD`, as that's the pattern we follow for child resource tests

Reply:
It is. See line #100.